src/components/pressables/Button.tsx

```diff
@@ -23,21 +23,27 @@ function Button(props: Props) {
   const isPrimaryOutline = variant === 'primary' && type === 'outline'
   const isSecondarySolid = variant === 'secondary' && type === 'solid'
   const isSecondaryOutline = variant === 'secondary' && type === 'outline'
+  const isTertiarySolid = variant === 'tertiary' && type === 'solid'
+  const isTertiaryOutline = variant === 'tertiary' && type === 'outline'
 
   const buttonStyles = cn({
-    'flex h-12 flex-col items-center justify-center rounded-full px-10 py-3 shadow': type === 'solid',
-    'flex h-12 flex-col items-center justify-center rounded-full px-10 py-3 shadow border border-solid': type === 'outline',
-    'flex h-12 flex-col items-center justify-center px-10 py-3': type === undefined || null,
+    'flex h-12 flex-col items-center justify-center rounded-full px-10 py-3 shadow': type === 'solid' && variant !== 'tertiary',
+    'flex h-12 flex-col items-center justify-center rounded-full px-10 py-3 shadow border border-solid': type === 'outline' && variant !== 'tertiary',
+    'flex h-12 flex-col items-center justify-center px-10 py-3': type === undefined || null && variant !== 'tertiary',
+    'flex px-4 py-1 items-center rounded-full': isTertiarySolid,
+    'flex px-4 py-1 items-center rounded-full border border-solid': isTertiaryOutline,
     'bg-[#7E5BFF]': isPrimarySolid && !disabled,
     'border-[#7E5BFF]': isPrimaryOutline && !disabled,
     'bg-[#F9F8FF33]': isPrimarySolid && disabled,
     'bg-[#F9F8FF]': isSecondarySolid && !disabled,
     'border-[#F9F8FF]':isSecondaryOutline,
     'bg-[#F9F8FF99]': isSecondarySolid && disabled,
+    'bg-[#9F85FF40]': isTertiarySolid,
+    'border-[#9F85FF40]': isTertiaryOutline,
   })
 
   const labelStyles = cn({
-    'text-12 text-white': (isPrimarySolid && !disabled) || (isSecondaryOutline && !disabled),
+    'text-12 text-white': (isPrimarySolid && !disabled) || (isSecondaryOutline && !disabled) || (variant === 'tertiary' && type !== undefined || null),
```

Comment:
does this need to use the isTertiary checks?

Reply:
thought it would be pointless to do a const `isTertiarty === 'tertiary'`

I could do a check thinking about it as hasNoType

Reply:
Perhaps would this be better with the following checks:
- isPrimary
- isSecondary
- isTertiary
- isOutline
- isSolid

And then we can build conditionals with those? My thought is just so these two blocks of classes is as readable as possible